src/OceanSeaIceModels/ocean_sea_ice_model.jl

```diff
@@ -76,21 +76,26 @@ initialize_jld2_file!(filepath, init, jld2_kw, including, outputs, model::OSIM)
 
 write_output!(c::Checkpointer, model::OSIM) = write_output!(c, model.ocean.model)
 
-function set_clock!(sim)
-    sim.model.clock.iteration = sim.model.ocean.model.clock.iteration
-    sim.model.clock.time = sim.model.ocean.model.clock.time
+function set_clock!(sim::PrescribedAtmosphere, time, iter)
+    sim.clock.time = time
+    sim.clock.iteration = iter
+    return nothing
+end
 
-    # Setting the atmosphere time to the ocean time
-    sim.model.atmosphere.clock.iteration = sim.model.ocean.model.clock.iteration
-    sim.model.atmosphere.clock.time = sim.model.ocean.model.clock.time
+function set_clock!(sim::OSIMSIM, time, iter)
+    sim.model.clock.time = time
+    sim.model.clock.iteration = iter
     return nothing
 end
 
 function set!(sim::OSIMSIMPA, pickup::Union{Bool, Integer, String})
     checkpoint_file_path = checkpoint_path(pickup, sim.output_writers)
 
     set!(sim.model.ocean.model, checkpoint_file_path)
-    set_clock!(sim)
+
+    iteration, time = sim.model.ocean.model.clock.iteration, sim.model.ocean.model.clock.time
```

Comment:
we will want `set_clock` for the ocean simulation too at some point

Reply:
yes, should we add now?